Assignment-2/index.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+require __DIR__ . '/Mail.php';
+
+if(isset($_POST['email'])) {
+  $mail = new Mail();
+  $mail->sendMail($_POST['email']);
+}
+?>
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>PHP Adv assignment-2</title>
+</head>
+<body>
+  <form action="./index.php" method="POST">
```

Comment:
I think just mentioning `action="/"` is enough.
Or there is no need to mention action at all maybe.

Reply:
Yes.